ui/src/automation.ts

```diff
@@ -2,6 +2,9 @@ import { dablHostname, deploymentMode, DeploymentMode, httpBaseUrl, ledgerId } f
 import { computeCredentials } from './Credentials';
 
 export const TRIGGER_HASH = process.env.REACT_APP_TRIGGER_HASH;
+
+// Do we want to keep this?
+// ts-prune-ignore-next
 export const EXBERRY_HASH = process.env.REACT_APP_EXBERRY_HASH;
```

Comment:
I've noticed here (and in other places throughout) there are some vars that are unused but look like we might use in the future. 

So I just left comments on them instead of deleting, if you see one that you think we can delete let me know!